cmd/identity/main.go

```diff
@@ -57,7 +57,7 @@ var (
 		Concurrency    uint   `default:"4" help:"number of concurrent workers for certificate authority generation"`
 		ParentCertPath string `help:"path to the parent authority's certificate chain"`
 		ParentKeyPath  string `help:"path to the parent authority's private key"`
-		Signer         certificates.CertClientConfig
+		Signer         certificatesclient.ClientConfig
```

Comment:
Why separate package? why not just `certificates.ClientConfig`?

Reply:
https://github.com/storj/docs/blob/master/code/Endpoint.md#client

Reply:
I see. Most probably `peer` related things should go to `certificates` in root dir (like `versioncontrol`). Then client can stay in `pkg/certificates`. @egonelbre

Reply:
I hadn't considered that.